tank/tflite/albert.py

```diff
@@ -0,0 +1,33 @@
+# RUN: %PYTHON %s
+import numpy as np
+from shark.shark_importer import SharkImporter
+import pytest
+
+model_path = "https://tfhub.dev/tensorflow/lite-model/albert_lite_base/squadv1/1?lite-format=tflite"
+
+# Inputs modified to be useful albert inputs.
+def generate_inputs(input_details):
+    for input in input_details:
+        print("\t%s, %s", str(input["shape"]), input["dtype"].__name__)
+
+    args = []
+    args.append(
+        np.random.randint(low=0, high=256, size=input_details[0]["shape"], dtype=input_details[0]["dtype"]))
+    args.append(np.ones(shape=input_details[1]["shape"], dtype=input_details[1]["dtype"]))
+    args.append(np.zeros(shape=input_details[2]["shape"], dtype=input_details[2]["dtype"]))
+    return args
+
+if __name__ == '__main__':
+    my_shark_importer = SharkImporter(model_path=model_path,
+                                      model_type="tflite",
+                                      model_source_hub="tfhub",
+                                      device="cpu",
+                                      dynamic=False,
+                                      jit_trace=True
+                                      )
+    input_details, output_details = my_shark_importer.get_model_details()
+    inputs = generate_inputs(input_details) # device_inputs
+    my_shark_importer.setup_inputs(inputs)
+    my_shark_importer.compile()
+    shark_results = my_shark_importer.forward()
```

Comment:
Can we set forward to require an input?

Reply:
this one we can set for "real" data inputs. The setup_input which we will incorporate into class init, we can just set as "dummy" input for shape info,

Reply:
can we feed the real "data" in this forward function, I think it'd be more intuitive this way :)

Reply:
I don't know where to find the real input of Albert model. If every model we try to find real data, it will take some time to read the every paper and find the dataset. I don't think it's that important right now.

Reply:
It doesn't make sense that the sharkinference.forward() takes input again, because in the constructor, you have set the input, and in compile(), you have used that input.